run_action.py

```diff
@@ -174,28 +174,63 @@ def main():
 
     github_token = os.environ.get("INPUT_GITHUB_TOKEN")
     github_api_url = os.environ.get("GITHUB_API_URL")
+    
     request_url = "%s/repos/%s/issues/%s/comments" % (
         github_api_url,
         repo,
         args.pull_request_id,
     )
 
+    headers = {
+        "Authorization": "token %s" % github_token,
+    }
+    report = {
+        "body": message
+    }
+
+    one_comment = os.environ.get("INPUT_ONE_COMMENT")
+    is_update = False
+    if one_comment.lower() in ("true", "1"):
```

Comment:
1. Let's default the environment variable to `false` for greater flexibility, i.e. not depend on GitHub Actions too much
2. Let's rename `is_update` to `comment_updated`
3. Let's evaluate things directly to a boolean, named `update_existing_comment`

```suggestion
    update_existing_comment = os.environ.get("INPUT_ONE_COMMENT", "false") in ("true", "1")
    comment_updated = False
    if update_existing_comment:
```